### ISSUE TITLE
Support i18n?

### ISSUE THREAD

**Issue by calfzhou** [Pelican-Elegant/elegant#96]
It will make pelican more widely useful if it can easily change its display language.

 [Jinja2 i18n](http://jinja.pocoo.org/docs/templates/#i18n)

**Comment by silverhook** [comment]
@calfzhou, did you manage to resolve this issue in your fork already?

**Comment by calfzhou** [comment]
@silverhook No, I did't work on it. I only tried to let blog posts support multiple translations.

**Comment by silverhook** [comment]
@calfzhou Hmmm, any idea what is preventing it from happening? Is it a Pelican-wide issue or could it be resolved when we re-write Elegant to get rid of Bootstrap?

**Comment by calfzhou** [comment]
I just haven't got chance to dive into it, since my blog not requires i18n.

Now we are moving forward, we could reprioritize our tasks.

**Comment by silverhook** [comment]
Very much agreed that we need to reprioritise tasks.

Can you suggest (perhaps in a separate issue) what you think would make sense? You have the most experience with Elegant both in the past, and especially right now as your fork is the most actively maintained.
